src/rendering/SceneManager.cc

```diff
@@ -1593,6 +1597,16 @@ void SceneManager::RemoveEntity(Entity _id)
     auto it = this->dataPtr->visuals.find(_id);
     if (it != this->dataPtr->visuals.end())
     {
+      // Remove visual's original transparency from map
+      rendering::VisualPtr vis = this->dataPtr->visuals[_id];
```

Comment:
`rendering::VisualPtr vis = it->second` to avoid another lookup

Reply:
[`9b15b6a`](https://github.com/ignitionrobotics/ign-gazebo/pull/878/commits/9b15b6a3e695a676cc06c47fd5a86af0766f64c9)